package.json

```diff
@@ -44,7 +44,7 @@
     "serve": "*",
     "sinon": "1.17.7",
     "unexpected": "10.23.0",
-    "unexpected-documentation-site-generator": "^4.2.3",
-    "unexpected-markdown": "1.4.1"
+    "unexpected-documentation-site-generator": "4.5.0",
```

Comment:
I think @sunesimonsen wanted the lax version range so plugins get the latest style updates whenever they are added to the site generator :)